packages/frontend/src/components/bill/NewBillCard.tsx

```diff
@@ -0,0 +1,150 @@
+import { Button, Input, Textarea } from '@nextui-org/react';
+import DatePicker from 'react-datepicker';
+import 'react-datepicker/dist/react-datepicker.css';
+import React, { ChangeEventHandler, useEffect, useState } from 'react';
+
+const FlatMates = [
+  {
+    name: 'Samuel',
+  },
+  {
+    name: 'David',
+  },
+  {
+    name: 'Owen',
+  },
+  {
+    name: 'Owen',
+  },
+  {
+    name: 'Owen',
+  },
+  {
+    name: 'Owen',
+  },
+  {
+    name: 'Owen',
+  },
+  {
+    name: 'Owen',
+  },
+];
+
+interface NewBillCardProps {
+  title?: string;
+  detail?: string;
+  due?: Long;
+}
+
+const NewBillCard: React.FC<NewBillCardProps> = () => {
+  const [dueDate, setDueDate] = useState(new Date());
+  const [unixTime, setUnixTime] = useState(0);
+  const [totalCost, setTotalCost] = useState('');
+  const [title, setTitle] = useState('');
+  const [detail, setDetail] = useState('');
+  const [flatmateNum, setFlatmateNum] = useState(6);
+  const [splitCost, setSplitCost] = useState('');
+  const [isEvenlySplit, setIsEvenlySplit] = useState(false);
```

Comment:
Would prefer if this was either refactored into a custom hook, or most of these moved into a single useState with an object containing all these keys.